src/wallet/test/coinselector_tests.cpp

```diff
@@ -115,10 +115,11 @@ static bool EquivalentResult(const SelectionResult& a, const SelectionResult& b)
     return ret.first == a_amts.end() && ret.second == b_amts.end();
 }
 
-static bool equal_sets(CoinSet a, CoinSet b)
+/** Check if this selection is equal to another one. Equal means same inputs (i.e same value and prevout) */
+static bool EqualResult(const SelectionResult& a, const SelectionResult& b)
 {
-    std::pair<CoinSet::iterator, CoinSet::iterator> ret = mismatch(a.begin(), a.end(), b.begin());
-    return ret.first == a.end() && ret.second == b.end();
+    std::pair<std::set<CInputCoin>::iterator, std::set<CInputCoin>::iterator> ret = std::mismatch(a.GetInputSet().begin(), a.GetInputSet().end(), b.GetInputSet().begin());
+    return ret.first == a.GetInputSet().end() && ret.second == b.GetInputSet().end();
```

Comment:
In 22e3b5b63c46d2e2dc601a8ca061823e21ab7151: Why delete alias?
```suggestion
    std::pair<CoinSet::iterator, CoinSet::iterator> ret = std::mismatch(a.GetInputSet().begin(), a.GetInputSet().end(), b.GetInputSet().begin());
    return ret.first == a.GetInputSet().end() && ret.second == b.GetInputSet().end();
```

Reply:
Done